draft-demarco-oauth-status-attestations.md

```diff
@@ -301,7 +300,8 @@ encoding, for better readability:
     "exp": 1698834139,
     "jti": "6f204f7e-e453-4dfd-814e-9d155319408c",
     "credential_format": "vc+sd-jwt",
```

Comment:
```suggestion
```

Reply:
seems like the `typ` or `cty` claim might be used here as well.

Reply:
I don't see how having these claims would be useful. How would the Credential Issuer use this information?

Reply:
yes, let's reduce the information about the payload of the credential for an augmented privacy

Reply:
@OR13 I have asked to remove the credential format, since we give only the credential hash without disclosing any information about the credential